### DISCUSSION TITLE
Corrective lenses

### DISCUSSION THREAD

**Discussion by crispyastrologer** [CodeFHD/OptiCore#3] [General]
Hello,

I am trying to create accurate corrective lenses for eye wear and this add-on can theoretically generate corrective lenses. 
The problem is that I am not sure how to find specific radii for Myopia or Hyperopia for example or general value of stock lenses for corrective vision that I can use in the add-on. Any information on this?

**Comment by CodeFHD** [comment]
Hi,

unfortunately, I have no exprience with eyeglass design, so I don't know about reference values for this case.

What the addon is still missing is "toric" lenses, i.e. a kind of cylindrical lens with different curvatures in the two axes, which I read is used for people with astigmatism. I'll have a look soon about adding that option.

**Reply by crispyastrologer** [reply]
I see, no worries. It's still a really helpful add-on thanks a lot!
